pkgs/development/python-modules/pyparser/default.nix

```diff
@@ -0,0 +1,59 @@
+{ buildPythonPackage, fetchPypi, fetchpatch, python, stdenv, python3Packages }:
+
+let
+  # pyparser only builds with parse == 1.6.5
+  parse = buildPythonPackage rec {
```

Comment:
No "private" packages in `python-packages.nix` because that can cause collisions.

Reply:
So how to do this then? We need this specific version (we have a more recent version in nixpkgs, but we need an older one for the papis package (see comment)).